samples/exemplar/src/main/kotlin/com/squareup/exemplar/dashboard/frontend/FrontendIndexAction.kt

```diff
@@ -0,0 +1,35 @@
+package com.squareup.exemplar.dashboard.frontend
+
+import kotlinx.html.h1
+import misk.config.AppName
+import misk.turbo.buildHtml
+import misk.turbo.turbo_frame
+import misk.web.Get
+import misk.web.PathParam
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.dashboard.AdminDashboardAccess
+import misk.web.mediatype.MediaTypes
+import misk.web.v2.Wrapper
+import wisp.deployment.Deployment
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class FrontendIndexAction @Inject constructor(
+  @AppName private val appName: String,
+  private val deployment: Deployment,
+) : WebAction {
+  @Get("/{suffix:.*}")
+  @ResponseContentType(MediaTypes.TEXT_HTML)
+  @AdminDashboardAccess
+  fun get(@PathParam suffix: String?): String {
+    return buildHtml {
+      Wrapper("/app", "$appName frontend", deployment.isLocalDevelopment) {
```

Comment:
wrapper is possibly a bad name, also passing in deployment might be weird too

Reply:
Wrapper builds the main HTML and <head> structure for the returned page. Maybe something-Layout, BaseLayout...etc would be more descriptive?

`deployment.isLocalDevelopment` is used to enable using the Tailwinds Play CDN in local development instead of the production minified tree-shaken version checked into the repo.